scrapy/core/downloader/handlers/http11.py

```diff
@@ -31,7 +31,9 @@
 class HTTP11DownloadHandler(object):
     lazy = False
 
-    def __init__(self, settings):
+    def __init__(self, crawler):
```

Comment:
It looks easy enough to make this "almost" backwards compatible: keep settings argument, add crawler=None argument.